### PR TITLE
Add PicPay as adopter

### PR DESCRIPTION
**Pull request by lcsbaroni**
Signed-off-by: Luís Baroni <luis.baroni@picpay.com>

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by cdiogo — APPROVED** [review]

**Review by RuanNunes — APPROVED** [review]

File: ADOPTERS.md [hunk 1]
```diff
@@ -64,6 +64,11 @@
...
+| [PicPay](https://www.picpay.com)                   | [Luis Baroni](https://github.com/lcsbaroni), [Renata Poluceno](https://github.com/renatapoluceno), [PicPay](https://github.com/picpay)                                                | Developer portal for building services throught templates, service catalog with ownership of services, documentation and metrics providing autonomy and visibility for all.                                                                                                                                                                                                                        |
+>>>>>>> Add PicPay as adopter
```
Comment (Rugvip) [inline ADOPTERS.md:74]: Looks like a bit of a merge conflict here that needs fixing 😅

**Review by Rugvip — COMMENTED** [review]
Thank you, and welcome! 🎉

**Review by eibrunorodrigues — APPROVED** [review]

File: ADOPTERS.md (hunk 1 above)
Comment (lcsbaroni) [reply ADOPTERS.md]: Sorry. I forgot a CTRL+S 🤦‍♂️.

**Review by Rugvip — APPROVED** [review]
:shipit:

**Review by rcastardo — APPROVED** [review]

**Pull request merged**